src/command/upload.ts

```diff
@@ -0,0 +1,134 @@
+import { Option, LeafCommand, Argument } from 'furious-commander'
+import { RootCommand } from './root-command'
+import * as FS from 'fs'
+import * as Path from 'path'
+import { sleep } from '../utils'
+import { Tag } from '@ethersphere/bee-js/dist/types'
+import { SingleBar, Presets } from 'cli-progress'
+import { bold, green, dim, red } from 'kleur'
+
+export class Upload extends RootCommand implements LeafCommand {
+  // CLI FIELDS
+
+  public readonly name = 'upload'
+
+  public readonly description = 'Upload file to Swarm'
+
+  @Argument({ key: 'path', describe: 'Path of the file (even directory)', demandOption: true })
+  public path!: string
+
+  // @Option({ key: 'encrypt', describe: 'Encrypt the passed data with a randomly generated key' })
```

Comment:
I agree, but I did because until the PR merge I would like to discuss whether currently we need this feature or not.

Reply:
I wouldn't enable for now because for uploading websites it's not necessary and for other use-cases it's easy to misuse. For example when uploading to a public gateway one may think that the data is privately encrypted. However since the encryption happens on the gateway, the owner of the gateway has access to the encryption key as well.